Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4631,6 +4632,50 @@ def com_google_fonts_check_varfont_instance_names(ttFont):
     yield PASS, "Instance names are correct"
 
 
+@check(
+  id = 'com.google.fonts/check/varfont_duplicate_instance_names',
+  rationale = """
+    According to Google Font’s specs (as of 2020-06-26), only weight and italic tokens are allowed in instance names (= "ExtraBold Italic").
+
+    This variable font contains duplicate instance names, suggesting that instances are defined for more then one axis.
+
+    Instead, only a base set of fonts for the most general representation of the family can be defined through instances in the `fvar` table,
+    all other instances will have to be left to access through the `STAT` table.
+  """,
```

Comment:
I'd like to get a better / clearer rationale description for this check. It mentions "only weight and italic tokens are allowed" but it is completely unrelated to the actual checking routine.

It also has wording like "This variable font contains duplicate instance names..." which sounds like a log message. A rationale description would read more like "This check's purpose is to detect duplicate named instance names in a given variable font"

It should also mention why this is bad / what's the benefit of fixing the issue (or what's the damage of not fixing it)

The rest of the rationale text currently also seem completely unrelated to the check so it should be completely rewritten from scratch.

Reply:
I changed it, but it still contains a clear hint at what the problem might be. A clear solution indication help me as a user to understand and solve the problem quickly.